active_directory/pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [build-system]
 requires = [
     "hatchling>=0.11.2",
-    "setuptools>=66; python_version > '3.0'",
```

Comment:
@ofek  do you remember why we kept setuptools as a dependency in Py3? Do we need it when we drop py2 completely? The jira card says yes, but the fact that it's explicitly listed as a dependency for Py3 makes me suspicious.